Cargo.toml

```
@@ -38,21 +41,20 @@ url = "2"
 byteorder = "1"
 tar = "0.4"
 tokio = { version = "1", features = ["full"] }
```

do we still need full feature tokio?

---

perhaps

```suggestion
tokio = { version = "1", features = ["time", "sync", "fs", "net"] }
```
https://docs.rs/crate/tokio-stream/0.1.12/features